pom.xml

```diff
@@ -3,31 +3,123 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <modelVersion>4.0.0</modelVersion>
+
     <groupId>com.redhat.cloud.notifications</groupId>
-    <artifactId>notifications-backend</artifactId>
+    <artifactId>notifications-parent</artifactId>
```

Comment:
Is there any reason why this pom still contains dependencies?

Reply:
Same reason as with the properties. They won't be used in the aggregator project.

Reply:
Ok but we're talking about the parent pom here. The dependencies should only be in the child pom, right?

Reply:
Yes, I moved them in the child pom as well. Sorry.. I am getting confused of it.

Reply:
I can still see many dependencies at parent level on the branch. I don't think the parent pom should contain any of them.